terra-jupyter-r/Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY scripts $JUPYTER_HOME/scripts
 
 # Add env vars to identify binary package installation
 ENV TERRA_R_PLATFORM="terra-jupyter-r"
-ENV TERRA_R_PLATFORM_BINARY_VERSION=3.13.0
+ENV TERRA_R_PLATFORM_BINARY_VERSION=3.14.0
```

Comment:
we should also update the [readme](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-r/README.md#image-contents) for this dockerfile to reflect the new version of bioconductor

Reply:
Good point. I need to also update this in the smoke tests I think.

Reply:
Done.